community/openvpn/sources

```diff
@@ -0,0 +1 @@
+https://github.com/OpenVPN/openvpn/archive/v2.4.8.tar.gz
```

Comment:
Try this link: `https://swupdate.openvpn.org/community/releases/openvpn-2.4.8.tar.xz`

This should remove the `autoreconf` requirement.

Reply:
cool!